Documentation/docs/learn/python_quick_start.md

```diff
@@ -0,0 +1,495 @@
+# Python Quick Start
+
+(quick-start)=
+
+## Installation
+
+To install the ITK Python package:
+
+```
+$ pip install itk
+```
+
+## Usage
+
+### Basic example
+
+Here is a simple python script that reads an image, applies a median image filter (radius of 2 pixels), and writes the resulting image in a file.
+
+```py
+#!/usr/bin/env python3
+
+import itk
+import sys
+
+input_filename = sys.argv[1]
+output_filename = sys.argv[2]
+
+image = itk.imread(input_filename)
+
+median = itk.median_image_filter(image, radius=2)
+
+itk.imwrite(median, output_filename)
+```
+
+### ITK and NumPy
+
+A common use case for using ITK in Python is to mingle NumPy and ITK operations on raster data. ITK provides a large number of I/O image formats and several sophisticated image processing algorithms not available in any other packages. The ability to intersperse that with the SciPy ecosystem provides a great tool for rapid prototyping.
+
+The following script shows how to integrate NumPy and `itk.Image`:
+
+```py
```

Comment:
Same comment as above

Reply:
Also, are we going to put the `#!/usr/bin/env python3` shebang in all of the examples? I would not put them for the sake of an easier reading.

Reply:
I agree -- for readability it is better to omit.